src/command/src/lib.rs

```diff
@@ -113,7 +143,17 @@ impl CommandExt for Command {
         self.stdout(Stdio::inherit()).stderr(Stdio::inherit())
     }
 
-    fn display(&mut self) -> String {
+    fn log(&mut self) -> &mut Self {
+        debug!("Command> {:?}", self.display());
+        self
+    }
+
+    fn print(&mut self) -> &mut Self {
+        println!("Command> {:?}", self.display());
+        self
+    }
+
+    fn display(&self) -> String {
```

Comment:
this changes API.  So need to bump up to 0.3?

Reply:
It doesn't edit any existing API, just adds new. So I think bump to `0.2.1` should be good